src/platform/datapath_raw_xdp_win.c

```diff
@@ -1735,6 +1918,22 @@ CxPlatXdpTx(
         Queue->Error = TRUE;
     }
 
+    if (XskRingOffloadChanged(&Queue->TxRing) &&
+        Queue->OffloadStatus.Transmit.ChecksumOffloadExtensions) {
+        uint32_t TxChecksumOffload;
+        QUIC_STATUS Status = GetTxOffloadConfig(Queue, &TxChecksumOffload);
+        if (QUIC_SUCCEEDED(Status)) {
+            Queue->OffloadStatus.Transmit.ChecksumOffload = !!TxChecksumOffload;
```

Comment:
If this gets enabled, should we be setting `XSK_SOCKOPT_TX_OFFLOAD_CHECKSUM` now?

Reply:
No, it's too late.

Reply:
https://github.com/microsoft/xdp-for-windows/blob/48117ff9c7322816c3393a2793c617c21f2837f3/published/external/afxdp_experimental.h#L80-L95